docs/source/onnxruntime/quantization.mdx

```diff
@@ -14,6 +14,84 @@ specific language governing permissions and limitations under the License.
 
 🤗 Optimum provides an `optimum.onnxruntime` package that enables you to apply quantization on many model hosted on the 🤗 hub using the [ONNX Runtime](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/tools/quantization/README.md) quantization tool.
 
+## Create `ORTQuantizer` class
```

Comment:
Not sure what tense is preferred in the docs, but I think in `transformers` we usually write the titles as follows:

```suggestion
## Creating an `ORTQuantizer`
```